build.gradle

```diff
@@ -0,0 +1,40 @@
+buildscript {
+    repositories {
+        jcenter()
+        maven { url 'http://repo.spring.io/plugins-release' }
+    }
+
+    dependencies {
+        classpath 'org.springframework.build.gradle:propdeps-plugin:0.0.7'
+    }
+}
+
+plugins {
+    id 'java'
+    id 'idea'
+    id 'maven'
+}
+
+description = 'Provides example for app with bunch of bugs to be testable'
+
+allprojects {
+    apply plugin: 'propdeps'
+    apply plugin: 'propdeps-maven'
+    apply plugin: 'propdeps-idea'
+
+    group = 'ru.yandex.autoschool'
+    version = '1.0-SNAPSHOT'
+
+    sourceCompatibility = 1.8
+    targetCompatibility = 1.8
+    buildDir = 'target'
```

Comment:
i've copied JS directly to `target` folder, so this change simplified migration. 
If i can find way to specify `${project.buildDir}` as argument for webpack, i'll omit this change